account_invoice_merge_attachment/models/account_move.py

```diff
@@ -4,13 +4,14 @@
 from odoo import models
 
 
-class AccountInvoice(models.Model):
+class AccountMove(models.Model):
 
-    _inherit = "account.invoice"
+    _inherit = "account.move"
 
-    @api.multi
-    def do_merge(self, keep_references=True, date_invoice=False):
-        invoices_info = super(AccountInvoice, self).do_merge(
+    def do_merge(
+        self, keep_references=True, date_invoice=False, remove_empty_invoice_lines=True
+    ):
+        invoices_info = super().do_merge(
```

Comment:
you should maybe write remove_empty_invoice_lines=remove_empty_invoice_lines when calling super ?

Reply:
:+1: